src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -215,6 +215,22 @@
     </PropertyGroup>
   </Target>
 
+  <PropertyGroup>
+    <GetNuGetPackageVersionsDependsOn>$(GetNuGetPackageVersionsDependsOn);CreateVersionInfoFile</GetNuGetPackageVersionsDependsOn>
+  </PropertyGroup>
+  
+  <Target Name="CreateVersionInfoFile"
+          DependsOnTargets="GetLatestCommitHash">
+    <PropertyGroup>
+      <SyncInfoFile>$(PackagesBasePath)\version.txt</SyncInfoFile>
```

Comment:
Originally, I called it ".version", because it appeared to me that was what the CLI team was doing.  Then I decided that I didn't want to potentially collide with their file.  In addition, I'm fairly confident that devs file associations are properly setup for opening a text file, so they don't have to extract the file first if they decide they want to open the file in some other program whereas I don't know what devs do or would do with a .version file (it's probably not setup for any file association, which is fine).  It also, corresponded with the type of file we have for the eula agreement files.  All of that made me decide it was preferable to use a .txt file extension vs something else.